java/client/src/test/java/glide/api/models/TransactionTests.java

```diff
@@ -763,6 +764,10 @@ InfScoreBound.NEGATIVE_INFINITY, new ScoreBoundary(3, false), new Limit(1, 2)),
         results.add(Pair.of(FlushAll, buildArgs()));
         results.add(Pair.of(FlushAll, buildArgs(ASYNC.toString())));
 
+        transaction.flushdb().flushdb(ASYNC);
+        results.add(Pair.of(FlushDB, buildArgs()));
+        results.add(Pair.of(FlushDB, buildArgs(ASYNC.toString())));
```

Comment:
This test will fail if we run against 6.1 or prior

Reply:
Added a fix for `flushall` tests too!